src/core/zod-to-openapi.ts

```
@@ -1,11 +1,25 @@
+import type { ZodObject, ZodType } from "zod";
 
 export default function convertToOpenAPI(schema: ZodType<unknown>, isArray: boolean) {
   const result = zodToJsonSchema(isArray ? schema.array() : schema, {
     target: "openApi3",
     $refStrategy: "none",
-  });
-  return result as SchemaObject;
+  }) as SchemaObject;
+  if (result.type === "object" && result.properties) {
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    for (const [propName, prop] of Object.entries<ZodType>((schema as ZodObject<{}>).shape)) {
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)